plugins/splunk-on-call/src/components/TriggerDialog/TriggerDialog.tsx

```diff
@@ -0,0 +1,365 @@
+                      const element = users.find(
+                        user => user.username === selectedUser,
+                      );
+                      return (
+                        <Chip
+                          key={selectedUser}
+                          label={`${element?.firstName} ${element?.lastName}`}
+                          className={classes.chip}
+                        />
+                      );
+                    })}
+                  </div>
+                )}
+                MenuProps={MenuProps}
+              >
+                {users.map(user => (
+                  <MenuItem key={user.email} value={user.username}>
+                    {user.firstName} {user.lastName}
+                  </MenuItem>
+                ))}
+              </Select>
+            </FormControl>
+            <FormControl className={classes.formControl}>
+              <InputLabel>Select Teams / Policies</InputLabel>
+              <Select
+                id="policy-targets"
+                multiple
+                value={policyTargets}
+                onChange={handlePolicyTargets}
+                input={<Input />}
+                renderValue={selected => (
+                  <div className={classes.chips}>
+                    {(selected as string[]).map(selectedPolicy => {
+                      const element = policies?.find(
+                        policy => policy.policy.slug === selectedPolicy,
+                      );
+                      return (
+                        <Chip
+                          key={selectedPolicy}
+                          label={element?.policy.name}
+                          className={classes.chip}
+                        />
+                      );
+                    })}
+                  </div>
+                )}
+                MenuProps={MenuProps}
+              >
+                {!policiesError &&
+                  !policiesLoaading &&
+                  policies &&
+                  policies.map(policy => (
+                    <MenuItem
+                      key={policy.policy.slug}
+                      value={policy.policy.slug}
+                    >
+                      {policy.policy.name}
+                    </MenuItem>
+                  ))}
+              </Select>
+            </FormControl>
+          </div>
+        </div>
+        <Typography
+          style={{ marginTop: '1em' }}
+          color="textSecondary"
+          gutterBottom
+        >
+          Acknowledge Behavior
+        </Typography>
+        <FormControl className={classes.formControl}>
+          <Select
+            id="multi-responder"
+            value={isMultiResponderValue}
+            onChange={isMultiResponderChanged}
+            inputProps={{ 'data-testid': 'trigger-select-behavior' }}
+          >
+            <MenuItem value="1">
+              Stop paging after a single Acknowledge from an escalation policy
```

Comment:
Not sure the tool, but should this say
```suggestion
              Stop paging after a single Acknowledgement from an escalation policy
```
???

Reply:
...or maybe "Acknowledgement of an escalation policy"? there may be wording in the tool itself which you've used or could be reused here, it just read a bit funny to me.

Reply:
It is a wording used in Splunk On-Call but I can understand that it is not very clear.
I changed it to: `Stop paging after a single escalation policy or user has acknowledged`.